--- docker/patches/wishlist.original.twig	2026-07-16 14:26:14
+++ docker/patches/wishlist.modified.twig	2026-07-16 14:29:31
@@ -38,7 +38,8 @@
             if (json['success']) {
                 $('#alert').prepend('<div class="alert alert-success alert-dismissible"><i class="fa-solid fa-circle-exclamation"></i> ' + json['success'] + ' <button type="button" class="btn-close" data-bs-dismiss="alert"></button></div>');
 
-                $('#wishlist').load('index.php?route=account/wishlist.list&language={{ language }}');
+                const customerToken = new URLSearchParams(window.location.search).get("customer_token");
+                $("#wishlist").load("index.php?route=account/wishlist.list&language={{ language }}&customer_token=" + encodeURIComponent(customerToken));
             }
         },
         error: function(xhr, ajaxOptions, thrownError) {
